magstock/templates/regextra.html

```diff
@@ -120,8 +122,8 @@
     <div id="comingwith" class="form-group" style="display:none">
         <label class="col-sm-2 control-label">Who's staying with you?</label>
         <div class="col-sm-6">
-            <input type="text" name="coming_with" value="{{ attendee.coming_with }}" class="form-control" placeholder="Full legal name of your Tent Leader">
-            <textarea name="coming_with" class="form-control" placeholder="Full legal names of everyone in your group">{{ attendee.coming_with }}</textarea>
+            <input disabled type="text" name="coming_with" value="{{ attendee.coming_with }}" class="form-control" placeholder="Full legal name of your Tent Leader">
```

Comment:
This is https://github.com/magfest/magstock/pull/33, I'm not really sure how it made it into this PR also.